### PR TITLE
feat: add avatar component

### PR DESCRIPTION
**Pull request by vtrbo**
- add avatar component
- add avatar docs

### CODE REVIEW

**Comment by wangyewei** [comment]
looks like there is an extra space in the class name.:

you can use the `createCls` function to solve it:
```typescript
import { createCls, getPrefixCls } from '@ikun-ui/utils'

const prefixCls = getPrefixCls('avatar')
$: cnames = createCls(prefixCls, cls)
```
```html
<div class={cnames}></div>
```

**Comment by wangyewei** [comment]
> looks like there is an extra space in the class name:
> 
>you can use the `createCls` function to solve it:
> 
> ```ts
> import { createCls, getPrefixCls } from '@ikun-ui/utils'
> 
> const prefixCls = getPrefixCls('avatar')
> $: cnames = createCls(prefixCls, cls)
> ```
> 
> ```
> <div class={cnames}></div>
> ```

the same problem exists in the other pull requests.
cc/ @baiwusanyu-c

**Comment by vtrbo** [comment]
> looks like there is an extra space in the class name.:  you can use the `createCls` function to solve it:
> 
> ```ts
> import { createCls, getPrefixCls } from '@ikun-ui/utils'
> 
> const prefixCls = getPrefixCls('avatar')
> $: cnames = createCls(prefixCls, cls)
> ```
> 
> ```
> <div class={cnames}></div>
> ```

ok, i fix it.

File: docs/components/KAvatar.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+| icon   | `string`                                             | `''`    | The class name of the icon, following the unocss standard. |
+| src    | `string`                                             | `''`    | the source of the image for an image avatar..              |
+| srcSet | `string`                                             | `''`    | native attribute `srcset` of image avatar.                 |
+| alt    | `string`                                             | `''`    | native attribute `alt` of image avatar.                    |
+| fit    | `fill` / `contain` / `cover` / `none` / `scale-down` | `cover` | set how the image fit its container for an image avatar.   |
+| size   | `string` / `number`                                  | `50px`  | avatar size, the default unit is `px`.                     |
+| radius | `string` / `number`                                  | `4px`   | avatar radius, the default unit is `px`.                   |
+| cls    | `string`                                             | `''`    | Additional class for                                       |
```
Comment (baiwusanyu-c) [inline docs/components/KAvatar.md:63]: `attrs` and `cls` I want to be props that all components have

File: docs/components/KAvatar.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+<demo src="../../../../example/avatar/types.svelte" github='Avatar'></demo>
+
+## Fallback
+
+fallback when image load error.
+
+<demo src="../../../../example/avatar/fallback.svelte" github='Avatar'></demo>
+
+## Fit Container
```
Comment (baiwusanyu-c) [inline docs/components/KAvatar.md:46]: Add an icon avatar document, there is no other problem

**Review by baiwusanyu-c — CHANGES_REQUESTED** [review]

File: docs/components/KAvatar.md (hunk 2 above)
Comment (baiwusanyu-c) [reply docs/components/KAvatar.md:46]: 看到了 在 types 里..

**Review by baiwusanyu-c — APPROVED** [review]

**Pull request merged**